glass/galaxies.py

```diff
@@ -298,92 +298,3 @@ def gaussian_phz(
             trunc = trunc[(znew < lower) | (znew > upper)]
 
     return zphot
-
-
-def _kappa_ia_nla(  # noqa: PLR0913
```

Comment:
Removing this function as it would present a blocker to converting. Some of these functions I cannot find in the initial documentation, e.g. `rho_c0` https://github.com/search?q=repo%3Aglass-dev%2Fcosmology+%22rho_c0%22&type=code.

The function is also not tested and not being used, xref https://github.com/glass-dev/glass/pull/21.